Remarkable-Donkeys/bangazon
# Kb Allow developers to access ProductType and Product resource

# Steps To Test #
1. Pull Testing branch
1. Run server
1. If not already done, POST a customer -> see Customer.cs for required fields
------- ProductType testing ------- 
2. POST a ProductType -> see ProductType.cs for required fields
4. GET list of all ProductTypes
5. GET single ProductType (using ProductTypeId)
6. PUT to edit ProductType field (using ProductTypeId)
7. DELETE ProductType (using ProductTypeId)
------- Product testing ------- 
3. POST a Product (with CustomerId and ProductTypeId as foreign keys) -> see Product.cs for required fields
4. GET list of all Products
5. GET single Product (using ProductId)
6. PUT to edit Product field(s) (using ProductId)
7. DELETE Product (using ProductId)

[Hyperlink to ProductType ticket](https://github.com/Remarkable-Donkeys/bangazon/issues/5)
[Hyperlink to Product ticket](https://github.com/Remarkable-Donkeys/bangazon/issues/2)

### Description of what should happen when steps are followed ###
By following the steps above, tester will add, list all, list single, edit, and delete ProductType and Product from database.

**At least one person needs to approve pull request before merged to master**
